src/scripts/main.js

```diff
@@ -1,3 +1,131 @@
 'use strict';
 
 // write code here
+
+const table = document.getElementsByTagName('table');
```

Comment:
```suggestion
const table = document.querySelector('table');
```
There is only one table

Reply:
is not fixed